deepmd/dpmodel/utils/type_embed.py

```diff
@@ -234,5 +234,6 @@ def get_econf_tebd(type_map, precision: str = "default"):
         [electronic_configuration_embedding[kk] for kk in type_map],
         dtype=PRECISION_DICT[precision],
     )
+    econf_tebd /= econf_tebd.sum(-1, keepdims=True)  # do normalization
```

Comment:
Isn't it `keepdim`?

https://pytorch.org/docs/stable/generated/torch.Tensor.sum.html

Reply:
Oh, I realize it is NumPy